Align sun/moon icon in theme toggle with navbar items

delize opened the pull request:
This PR attempts to align the theme toggle selector with the rest of the navbar items, for a more professional look.

For example:
https://hugo-profile.netlify.app/
![Screenshot 2025-01-22 at 6 26 06 PM](https://github.com/user-attachments/assets/ef93d15d-51e4-4dc6-bd0a-8e356f75b92b)


To:

**Widescreen**
![Screenshot 2025-01-22 at 6 27 36 PM](https://github.com/user-attachments/assets/20dbbc17-f057-4373-847a-21cabc0807b8)

**Mobile / iPad**
<img width="794" alt="Screenshot 2025-01-22 at 6 28 23 PM" src="https://github.com/user-attachments/assets/8c14f2a0-62e4-4652-902c-49e668f09d0f" />


 Nothing wrong with the current implementation, but the off-set theme toggle really bothers me UI / UX wise.

jochavez committed:
updating logo image settings

jochavez committed:
update size

jochavez committed:
revert logo size

jochavez committed:
removing footer line

jochavez committed:
update links

jochavez committed:
update

jochavez committed:
update url

jochavez committed:
update url

jochavez committed:
revert url

VAISHAKH-GK committed:
Make whole html colour black

oscareduardolp6 committed:
Improve SEO changing HTML Tags at Hero section

oscareduardolp6 committed:
Add language attribute to HTML tag in baseof.html

delize committed:
Align sun/moon icon in theme toggle with navbar items

deining committed:
Fix error when running example site with latest hugo version 0.143.0

urban-adeininger committed:
Remove deprecated shortcode 'gist'
Fix broken anchor links

goldjg committed:
Rewrite search.js to harden against XSS

CeduDev commented on static/css/header.css:
```diff
@@ -12,11 +12,23 @@ header {
 
 header .navbar .navbar-nav a {
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Vertically align all items */
 }
```
I like this fix, definitely bothered me as well. However, with this implementation on mobile, the hamburger menu items are all aligned to the left (they were aligned to the center before). Personally, I like that aligned more, which thus would require this change 
```suggestion
header .navbar .navbar-nav a,
header .navbar .navbar-nav div {
  color: var(--text-color) !important;
  display: flex;
  align-items: center; /* Vertically align all items */
  justify-content: center /* Horizontally align all items */;
}
```

This ofc is up to @gurusabarish, however they see fit

delize replied:
Hey, not opposed to implementing this in the PR. Originally I only checked how this looked on Desktop, not mobile, but noticed it several weeks after the PR. I just was okay to live with it. When I'm back at a computer I'll see about trying to incorporate that change in to the PR.

CeduDev committed:
update gitignore

CeduDev committed:
fix social network links to use target='_blank'

delize committed:
Implement request by @CeduDev

XintongWang4869 committed:
docs: improve README structure and language, fix tweet shortcode

- Reorganized and rewrote README to improve flow and clarity for beginners
- Fixed broken `x` shortcode by replacing it with `tweet`

Note: Twitter embed may fail due to TLS handshake timeout.

XintongWang4869 committed:
docs: add content in Content Management section in README, and refine wording

gurusabarishkumar-a-17149 committed:
same company multiple experience support

gurusabarish commented on static/css/header.css:
```diff
@@ -12,11 +12,24 @@ header {
 
 header .navbar .navbar-nav a {
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Vertically align all items */
+    justify-content: center; /* Horizontally align all items */
 }
 
 #theme-toggle {
     background-color: var(--background-color) !important;
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Center the icons vertically */
+    justify-content: center; /* Center the icons horizontally */
+    width: 40px; /* Set consistent width */
+    height: 40px; /* Set consistent height */
+    border: none; /* Remove default border */
+    background: none; /* Remove default background */
+    cursor: pointer; /* Pointer for interactivity */
+    margin: 0; /* Reset margin */
+    padding: 0; /* Reset padding */
```
Kindly try to reuse bootstrap classes instead writing new css handling

delize replied:
Resolved as much as possible

gurusabarish commented on static/css/header.css:
```diff
@@ -12,11 +12,24 @@ header {
 
 header .navbar .navbar-nav a {
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Vertically align all items */
+    justify-content: center; /* Horizontally align all items */
 }
 
 #theme-toggle {
     background-color: var(--background-color) !important;
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Center the icons vertically */
+    justify-content: center; /* Center the icons horizontally */
+    width: 40px; /* Set consistent width */
+    height: 40px; /* Set consistent height */
+    border: none; /* Remove default border */
+    background: none; /* Remove default background */
+    cursor: pointer; /* Pointer for interactivity */
+    margin: 0; /* Reset margin */
+    padding: 0; /* Reset padding */
```
Kindly try to reuse bootstrap classes instead writing new css handling

delize replied:
Resolved as much as possible

delize committed:
Rebase and reuse bootstrap css

delize committed:
Merge branch 'master' into align-theme-toggle-in-navbar
